File: Day 12/Homework/task2.py
'''გააკეთეთ 5-5 მაგალითი while ციკლებზე'''
#1
i=1
while i<=5:
    print(i)
    i=i+1
#2
i=1
sum=0
while i<=5:
    sum=sum+i
    i=i+1
print(sum)
#3
i=1
numr=1
while i<=5:
    numr=numr*i
    i=i+1
print(numr)
#4
i=1
while i<=5:
    if i>3:
        break
    print(i)
    i=i+1
#5
i = 0
while i < 6:
    i=i+1
    if i == 3:
        continue
    print(i)

File: Day 186/Classwork/task2.js
const args = process.argv.slice(2); 

if (args.length < 3) {
  console.log("please enter 3 numbers");
} else {
  const numbers = args.map(Number);
  const sum = numbers.reduce((a, b) => a + b, 0);
  console.log(`sum of numbers: ${sum}`);
}
